contracts/factories/base-factory/Cargo.toml

```diff
@@ -35,5 +35,7 @@ schemars        = { workspace = true }
 serde           = { workspace = true }
 sg1             = { workspace = true }
 sg2             = { workspace = true }
+cw721-base = { workspace = true }
```

Comment:
```suggestion
cw721-base = { workspace = true, features = ["library"] }
```
Be sure to import as a library, not a full contract.

Reply:
updated